housekeeping/docker/kagome-dev/minideb.Dockerfile

```diff
@@ -23,10 +23,17 @@ RUN curl -fsSL https://download.docker.com/linux/debian/gpg | gpg --dearmor -o /
       bullseye stable" | tee /etc/apt/sources.list.d/docker.list > /dev/null && \
     echo \
       "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-11 main" | tee -a /etc/apt/sources.list.d/docker.list > /dev/null && \
+    echo \
+      "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-12 main" | tee -a /etc/apt/sources.list.d/docker.list > /dev/null && \
+    echo \
+      "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-13 main" | tee -a /etc/apt/sources.list.d/docker.list > /dev/null && \
     echo \
       "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-14 main" | tee -a /etc/apt/sources.list.d/docker.list > /dev/null && \
+    echo \
+      "deb http://apt.llvm.org/bullseye/ llvm-toolchain-bullseye-15 main" | tee -a /etc/apt/sources.list.d/docker.list > /dev/null && \
     echo \
       "deb http://deb.debian.org/debian/ testing main" | tee -a /etc/apt/sources.list.d/docker.list > /dev/null && \
+    wget -q -O- https://github.com/rui314/mold/releases/download/v1.3.1/mold-1.3.1-x86_64-linux.tar.gz | tar -C /usr/local --strip-components=1 -xzf - && \
```

Comment:
Probably would be better to install mold by apt-get

Reply:
Done